examples/step-68/doc/intro.dox

```diff
@@ -0,0 +1,119 @@
+is obtainedin two different fashions:
+- By evaluating the velocity function at the location of the particles.
+- By evaluating the velocity function on a background triangulation and, using
+a  finite element support, interpolating at the position of the particle.
+
+The first approach is not really practical, since in general, the velocity profile
+is not known analytically. The second approach, based on interpolating a solution
+at the position of the particles, mimics exactly what would be done in a
+realistic computational fluid dynamic simulation. In this step, we illustrate both strategies.
+
+
+We note that much greater accuracy could be obtained by using a fourth
+order Runge-Kutta method or another appropriate scheme for the time integration
+of the motion of the particles.
+
+<h3>Particles in deal.II</h3>
+
+In deal.II, Particles::Particle are very simple and flexible entities that can be used
+to build PIC, DEM or any type of particle-based models. Particles have a location
+in real space, a location in the reference space of the element in which they
+lie and a unique ID. In the majority of cases, simulations that include
+particles require a significant number of them. Thus, it becomes interesting
+to handle all particles through an entity which agglomerates all particles.
+In deal.II, this is achieved through the use of the Particles::ParticleHandler class.
+
+By default, particles do not have a diameter,
+a mass or any other physical properties which we would generally expect of physical particles. Howevever, through
+a ParticleHandler, particles have access to a Particles::PropertyPool. This PropertyPool is
+an array which can be used to store any  arbitrary number of properties
+associated with the particles. Consequently, users can build their own
+particle solver and attribute the desired properties to the particles (e.g. mass,
+  diameter, temperature, etc.). In the present tutorial, this is used to
+  store the value of the fluid velocity and the process id to which the particles
+  belong.
+
+
+<h3>The testcase</h3>
+
+In the present step, we use particles as massless tracer to illustrate
+the dynamic of a vortical flow : the Rayleigh-Kotte Vortex. This flow pattern
+is generally used as a complex test case for interface tracking methods (e.g.
+  volume-of-fluid and level set approches) since
+it leads to strong rotation and elongation of the fluid (Blais 2014).
+
+The
+stream function $\Psi$ of this Rayleigh-Kotte vortex is defined as:
+
+@f[
+\Psi = \frac{1}{\pi} sin^2 (\pi x) \sin^2 (\pi y) \cos \left( \pi \frac{t}{T} \right)
+@f]
+where $T$ is the period of the flow. The velocity profile in 2D ($\textbf{u}=[u,v]^T$) is :
+@f{eqnarray*}
+   u &=&  - \frac{\partial\Psi}{\partial y} = \frac{1}{\pi} \sin^2 (\pi x) \sin (\pi y) \cos (\pi y)  \cos \left( \pi \frac{t}{T} \right)\\
+   v &=&  \frac{\partial\Psi}{\partial x} = \frac{1}{\pi} \cos(\pi x) \sin(\pi x) \sin^2 (\pi y) \cos \left( \pi \frac{t}{T} \right)
```

Comment:
Its hard to read the formulas in text form, but is there a factor of 2pi missing in both u and v (from the outer and inner derivative of the product rule of `sin^2(pi y)` and `sin^2(pi x)` respectively)? Or is it missing in the stream function if the velocity should remain unchanged?